src/plugins/dashboard/public/application/actions/copy_to_dashboard_modal.tsx

```diff
@@ -67,73 +69,85 @@ export function CopyToDashboardModal({
   }, [dashboardOption, embeddable, selectedDashboard, stateTransfer, closeModal]);
 
   return (
-    <PresentationUtilContext>
-      <EuiModalHeader>
-        <EuiModalHeaderTitle>{dashboardCopyToDashboardAction.getDisplayName()}</EuiModalHeaderTitle>
-      </EuiModalHeader>
+    <EuiFocusTrap clickOutsideDisables={true}>
+      <EuiOutsideClickDetector onOutsideClick={closeModal}>
+        <div role="dialog">
+          <PresentationUtilContext>
+            <EuiModalHeader>
+              <EuiModalHeaderTitle>
+                {dashboardCopyToDashboardAction.getDisplayName()}
+              </EuiModalHeaderTitle>
+            </EuiModalHeader>
 
-      <EuiModalBody>
-        <>
-          <EuiText>
-            <p>{dashboardCopyToDashboardAction.getDescription()}</p>
-          </EuiText>
-          <EuiSpacer />
-          <EuiFormRow hasChildLabel={false}>
-            <EuiPanel color="subdued" hasShadow={false} data-test-subj="add-to-dashboard-options">
-              <div>
-                {capabilities.canEditExisting && (
-                  <>
-                    <EuiRadio
-                      checked={dashboardOption === 'existing'}
-                      data-test-subj="add-to-existing-dashboard-option"
-                      id="existing-dashboard-option"
-                      name="dashboard-option"
-                      label={dashboardCopyToDashboardAction.getExistingDashboardOption()}
-                      onChange={() => setDashboardOption('existing')}
-                    />
-                    <div className="savAddDashboard__searchDashboards">
-                      <DashboardPicker
-                        isDisabled={dashboardOption !== 'existing'}
-                        idsToOmit={dashboardId ? [dashboardId] : undefined}
-                        onChange={(dashboard) => setSelectedDashboard(dashboard)}
-                      />
+            <EuiModalBody>
+              <>
+                <EuiText>
+                  <p>{dashboardCopyToDashboardAction.getDescription()}</p>
```

Comment:
Assuming you can wire up the `aria-describedby={randomDescriptionId}` I showed above, you'd then add the `id` here

```suggestion
                  <p id={randomDescriptionId}>
                  	{dashboardCopyToDashboardAction.getDescription()}
                  </p>
```